From: Owen Synge <osynge@suse.com>
Date: Tue, 12 Aug 2014 10:00:08 +0200
Subject: [PATCH] Switch off systemd detection

Currently systemd on ceph breaks shutting down ceph.

(cherry picked from commit 0e8635704a62d53d4876f5f461e16ba84557f4f6)
---
 src/init-ceph.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/init-ceph.in b/src/init-ceph.in
index 5fb8d6d..6506c56 100644
--- a/src/init-ceph.in
+++ b/src/init-ceph.in
@@ -48,6 +48,8 @@ EXIT_STATUS=0
 # detect systemd
 SYSTEMD=0
 grep -qs systemd /proc/1/comm && SYSTEMD=1
+# dont detect on SLE12
+SYSTEMD=0
 
 signal_daemon() {
     name=$1
